README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
```

Comment:
Perhaps change this input key from the space bar to the enter key. The enter key may be a better key choice for starting the game, and this way the space bar's only purpose is to pause the game instead.